mlir/test/Dialect/ArmSME/outer-product-fusion.mlir

```diff
@@ -405,6 +1181,37 @@ func.func @outerproduct_widening_2way__bad_defining_op(
   return %1 : vector<[4]x[4]xf32>
 }
 
+// -----
+
+// CHECK-LABEL: @outerproduct_widening_4way__bad_defining_op
```

Comment:
Not sure I see what's "wrong" with this example :)

Reply:
same as above but for 4-way, i.e. Defining op of outer product must be a supported extension op.

But I've added a comment where it goes wrong to clarify.